### PR TITLE
update os/booting-on-interoute.md

### PR DESCRIPTION
**Pull request by phillipkent**
Please update the instructions for Interoute VDC.

- phillip.kent@interoute.com

### CODE REVIEW

File: os/booting-on-interoute.md [hunk 1]
```diff
@@ -1,196 +1,268 @@
 # Running CoreOS on Interoute VDC
 
-Interoute VDC is Europe's largest cloud services platform, which is powered by Apache Cloudstack.
+(Version date: 2015-12-23. Author: Phillip Kent for Interoute)
```
Comment (joshix) [inline os/booting-on-interoute.md]: Delete line - version/blame info is in source control metadata

File: os/booting-on-interoute.md [hunk 2]
```diff
@@ -1,196 +1,266 @@
 # Running CoreOS on Interoute VDC
 
-Interoute VDC is Europe's largest cloud services platform, which is powered by Apache Cloudstack.
+[Interoute Virtual Data Centre](https://cloudstore.interoute.com/what_is_vdc) is an Infrastructure-as-a-Service platform, which is integrated into Interoute's high-performance [global fibre optic network](https://cloudstore.interoute.com/networking).
```
Comment (joshix) [inline os/booting-on-interoute.md]: delete "high-performance"

File: os/booting-on-interoute.md [hunk 3]
```diff
@@ -1,196 +1,266 @@
...
+This document is a guide to deploying a single CoreOS virtual machine on Interoute VDC. The CoreOS default configuration of SSH keypair-based authentication will be used.
 
-* You have an Interoute VDC account. You can easily [sign up for a free trial](http://cloudstore.interoute.com/main/TryInterouteVDCFREE) (no credit card required).
-* You have the  Cloudmonkey command line tool installed and configured on the computer that you are working on. Instructions on how to install and configure Cloudmonkey so that it can communicate with the VDC API can be found in the [Introduction to the VDC API](http://cloudstore.interoute.com/main/knowledge-centre/library/vdc-api-introduction-api).
-* You have installed OpenSSH client software. This is usually already installed in Linux and Mac OS. For Windows it can be downloaded at the [OpenSSH website](http://www.openssh.com/).
+Note: commands beginning with '$' are to be typed into the command line, and commands beginning '>' are to be typed into Cloudmonkey. Long lines of code have been broken up for display purposes, indicated by a continuation symbol '\', but if you copy the code make sure to remove any linebreaks.
```
Comment (joshix) [inline os/booting-on-interoute.md]: Soft suggestion: "If your terminal has trouble with the escape character, try removing the documentation line breaks." 

(If we don't believe `\\n` works when pasted in most shells, we should probably not break the lines up in the first place.)

File: os/booting-on-interoute.md [hunk 4]
```diff
@@ -1,196 +1,266 @@
...
-* You have installed OpenSSH client software. This is usually already installed in Linux and Mac OS. For Windows it can be downloaded at the [OpenSSH website](http://www.openssh.com/).
+Note: commands beginning with '$' are to be typed into the command line, and commands beginning '>' are to be typed into Cloudmonkey. Long lines of code have been broken up for display purposes, indicated by a continuation symbol '\', but if you copy the code make sure to remove any linebreaks. 
 
-Note: In the following steps, commands beginning with '$' are to be typed into the command line, and commands beginning '>' are to be typed into Cloudmonkey. Long lines of code have been broken up for display purposes, indicated by a continuation symbol '\', but if you copy the code make sure to remove any linebreaks. 
+## Prerequisites
+
+The following are assumed:
+
+* You have an Interoute VDC account. You can easily [sign up for a free trial](https://cloudstore.interoute.com/vdc-trial).
+* You have got an API key and Secret key for your VDC account ([how to generate API access keys for VDC](http://cloudstore.interoute.com/knowledge-centre/library/vdc-api-introduction-api)).
```
Comment (joshix) [inline os/booting-on-interoute.md]: change to:
You have an API key and Secret key for your VDC account. See [how to generate API access keys for VDC](http://cloudstore.interoute.com/knowledge-centre/library/vdc-api-introduction-api) if you need to create these.

File: os/booting-on-interoute.md [hunk 5]
```diff
@@ -1,196 +1,266 @@
...
+* You have the  Cloudmonkey command line tool installed and configured for Interoute VDC on the computer that you are working on.
```
Comment (joshix) [inline os/booting-on-interoute.md]: delete extra space after "the"

Comment (joshix) [inline os/booting-on-interoute.md]: Would you like to add a link to download/install the Cloudmonkey tool here?

File: os/booting-on-interoute.md [hunk 6]
```diff
@@ -1,196 +1,266 @@
...
+> listServiceOfferings name=4096-2 filter=id
++--------------------------------------+
+|                  id                  |
++--------------------------------------+
+| 85228261-fc66-4092-8e54-917d1702979d |
++--------------------------------------+
 ```
 
-The keypair 'name' parameter is arbitrary and is used for your reference only.
+Finally, the 'networkids' parameter specifies the network(s) which the virtual machine will be attached to. You need to specify at least one network.
```
Comment (joshix) [inline os/booting-on-interoute.md]: "to which the virtual machine will be attached"

File: os/booting-on-interoute.md [hunk 7]
```diff
@@ -1,196 +1,266 @@
...
+The last configuration step is to create an egress firewall rule for the network so that the CoreOS virtual machine will be able to get outward access to the Internet. This is needed to allow CoreOS to access update servers to do automatic updating, and for Docker to access repositories for container images:
+
+```sh
+> createEgressFirewallRule networkid=c5841e7c-e69e-432b-878b-c108b07a160f protocol=all cidr=0.0.0.0/0
+```
 
-So finally the VM is set up for connection, using the 'ipaddress' found from the listPublicIpAddresses command and specifying the private SSH key file to match the public key which I registered in VDC. Type this ssh command into a terminal:
+(Note that these network rules are simple and permissive, while rules for production virtual machines should always be more strictly defined.)
 
+The virtual machine is now set up for connection, using the 'ipaddress' found above and specifying the private SSH key file to match the public key which was registered in VDC (and there is no root user in CoreOS, the default user is 'core'):
```
Comment (joshix) [inline os/booting-on-interoute.md]: Change to:
"The virtual machine is now set up for connection, using the `ipaddress` found above, and specifying the private SSH key file to match the public key which was registered in VDC. Note that there is no `root` user in CoreOS; the default user is named `core`:"

**Comment by joshix** [comment]
@phillipkent Thanks for these thorough updates. Sorry I had to interrupt my review yesterday. I have a few more comments and suggestions, and then I'll be happy to merge these changes.

File: os/booting-on-interoute.md (hunk 2 above)
Comment (phillipkent) [inline os/booting-on-interoute.md]: OK

File: os/booting-on-interoute.md (hunk 3 above)
Comment (phillipkent) [inline os/booting-on-interoute.md]: My colleague wrote this originally with linebreaks for clarity of the command structure. My personal preference would be to remove all the linebreaks and the commands are then directly executable.

File: os/booting-on-interoute.md (hunk 4 above)
Comment (phillipkent) [inline os/booting-on-interoute.md]: ok

File: os/booting-on-interoute.md (hunk 7 above)
Comment (phillipkent) [inline os/booting-on-interoute.md]: ok

**Comment by phillipkent** [comment]
Thanks Josh. I will make the changes above and re-submit. Feel free to make more edits for agreement with house style.

**Comment by joshix** [comment]
@phillipkent Thanks for making the changes. LGTM for merge. This will appear on the coreos/docs live site after next site sync.

**Pull request merged**